File: study/src/study/day27.java
package study;

import java.util.*;

public class day27 {
    public int[] solution(int []arr) {
        ArrayList<Integer> list = new ArrayList<>();
        for (int i=0; i<arr.length; i++){
            if (i == 0){
                list.add(arr[i]);
            }else{
                if (arr[i-1]!=arr[i]){
                    list.add(arr[i]);
                }
            }
        }
        int[] answer = list.stream().mapToInt(i -> i).toArray();
        return answer;
    }
}

File: study/src/study/day27.py
def solution(arr):
    answer = []
    for n in range(len(arr)):
        if n == 0:
            answer.append(arr[n])
        else:
            if(arr[n-1] != arr[n]):
                answer.append(arr[n])
    return answer

test = [1,1,3,3,0,1,1]
print(solution(test))